### ISSUE TITLE
[E2E] Huggingface GPTNeoForCausalLM and GPTNeoForSequenceClassification training accuracy failed

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#703]
### 🐛 Describe the bug

Model list:
- [ ] `GPTNeoForCausalLM`
- [ ] `GPTNeoForSequenceClassification`

WARNING:common:fp64 golden ref were not generated for GPTNeoForCausalLM. Setting accuracy check to cosine
WARNING:common:current_device=xpu; error:value cannot be converted to type float without overflow
W0802 18:11:51.918000 3841258 torch/_dynamo/utils.py:1499] Similarity score=0.9365086555480957
E0802 18:11:51.919000 3841258 torch/_dynamo/utils.py:1450] Accuracy failed for key name transformer.h.0.attn.attention.k_proj.weight.grad
fail_accuracy

### Versions

torch-xpu-ops: https://github.com/intel/torch-xpu-ops/commit/1d70431c072db889d9a47ea4956049fe340a426d
pytorch: d224857b3af5c9d5a3c7a48401475c09d90db296
device: pvc 1100, bundle: 0.5.3, driver: 803.61

**Comment by chuanqi129** [comment]
Those 2 models belongs to skip list before, commit https://github.com/pytorch/pytorch/commit/8458980bbf78714a0fbe703785c100cad523fade change the skip logic. We'll submit PR to skip them again, but we need to double check whether we have potential real issue in here.

**Comment by retonym** [comment]
low priority for not included in Meta PyTorch dashboard

**Comment by retonym** [comment]
This model is not included in Meta dashboard for A100. 
We could not track the condition of their condition, passing or not.
Propose to re-check this model status, once it is included in meta public dashboard.
